apps/zotonic_mod_zotonic_site_management/priv/templates/addsite.tpl

```diff
@@ -166,7 +166,7 @@
                             <div class="col-sm-7">
                                 <input class="form-control" type="text" id="{{ #dbuser }}" name="dbuser" value="" placeholder="{{ m.config.zotonic.dbuser.value|escape }}" />
                                 {% validate id=#dbuser name="dbuser"
-                                    type={format pattern="^[A-Za-z0-9_]*$" failure_message=_"Please use only a…z and 0…9 characters."}
+                                    type={format pattern="^[A-Za-z0-9_.]*$" failure_message=_"Please use only a…z and 0…9 and . characters."}
```

Comment:
Shouldn't that be `\\.` as otherwise it would accept any character, which the `.` does in regexps?

Reply:
Never mind - your solution is correct 👍